server/controllers/user.controller.js

```diff
@@ -6,7 +8,8 @@ import User from '../models/user.model';
 function load(req, res, next, id) {
   User.get(id)
     .then((user) => {
-      req.user = user; // eslint-disable-line no-param-reassign
+      user.password = null;
```

Comment:
This should be done at the model level. with   `password: { type: String, select: false }`
and including should be explicit: `select('+password')`

Reply:
Should actually be deleting this property from the object instead.